### DISCUSSION TITLE
Reopen the data viewer

### DISCUSSION THREAD

**Discussion by fretwurst** [posit-dev/positron#4191] [Feedback] [answered]
When I click on the table icon in VARIABLES for a loaded dataset, the data viewer opens as expected. However, when I then work on other files (data viewer open in the background) and then click on the table icon again, nothing happens—the data viewer does not open the same data in a new viewer (as it does with View(data)) or bring the already open data viewer to the foreground like in R-Studio. 

[Imagine that I click on the table icon. You can just barely see the tab for Data in the top right next to the Preview icon. If I open a qmd file further to the left, you can't even see that, and nothing happens when you click on the table icon.]
<img width="1512" alt="Bildschirmfoto 2024-07-31 um 10 37 36" src="https://github.com/user-attachments/assets/e5e5549a-5c22-4f8b-92f4-6ad09be22b48">

**Comment by dfalbel** [comment, marked as answer]
Thanks for letting us know. This indeed looks like a bug. I have opened https://github.com/posit-dev/positron/issues/4197#issue-2440554992 to track this.
